generated/python/grpc-google-iam-v1/PUBLISHING.rst

```diff
@@ -0,0 +1,29 @@
+PUBLISHING
+----------
+
+Note: This folder has been generated by the gen-api-package tool.
+
+The instructions assumes that no changes have been made to the folder and its
+contents since it was created.
+
+PREREQUISITES
+-------------
+
+- Python must installed
```

Comment:
Typo. No need to regenerate, but can you do a quick fix in the packman template?